### ISSUE TITLE
"compact" always active

### ISSUE THREAD

**Issue by SeimenEll** [Arth-ur/sphinxcontrib-bitfield#2]
Hi,

I try to generate a old documentation with a new installation. For some reason the "compact" flag seems to be active all the time. Do you have a idea, what's wrong?

My bitfield.rst:

```
.. bitfield::
    :bits: 32
    :lanes: 1

    [
       { "name": "Read and Write", "bits": 8, "type": 5 },
       { "name": "Read only", "bits": 8, "type": 4 },
       { "name": "Write only", "bits": 8, "type": 3 },
       { "name": "Reserved", "bits": 8, "type": 1 }
    ]

```

Output:
![image](https://user-images.githubusercontent.com/34028124/168075753-1b763a35-378b-4f79-98e9-849893fb0dc8.png)

Versions:
Sphinx 1.8.5
sphinxcontrib-bitfield 0.4.0
bit-field 0.4.0

If i run 

`bit_field test.json > test.svg`

i get this

![image](https://user-images.githubusercontent.com/34028124/168076774-e954f19c-0b8c-4dc6-8149-dbe2a84c5136.png)

test.json:

```
    [
       { "name": "Read and Write", "bits": 8, "type": 5 },
       { "name": "Read only", "bits": 8, "type": 4 },
       { "name": "Write only", "bits": 8, "type": 3 },
       { "name": "Reserved", "bits": 8, "type": 1 }
    ]
```

**Comment by Arth-ur** [comment]
Hello, 

there is an issue with the Latex builder that should be fixed by b4342142.

Can you check that the new release [sphinxcontrib-bitfield v0.4.1](https://pypi.org/project/sphinxcontrib-bitfield/0.4.1/) works for you?

Thanks!

**Comment by SeimenEll** [comment]
It does work, thank you very much.
